src/styles/mimetypes.styl

```diff
@@ -0,0 +1,11 @@
+/*
+Generates a list of rules like:
+.my-prefix-audio
+    background-image embedurl('path/to/icon-type-audio.svg')
+
+It's a quick way to generate all the selectors for the different file types
+*/
+mime-types(selector-prefix, icons-path)
```

Comment:
Smart move. 🥇 
Although I'm afraid it will break on binaries and spreadsheets as the types are named `bin`& `sheet` but the svgs are named "binary" and "spreadsheet".